ocs_ci/ocs/resources/ocs.py

```diff
@@ -212,6 +212,12 @@ def get_ocs_csv():
         CSVNotFound: In case no CSV found.
 
     """
+
+    odf_csv = OCP(kind="operator", resource_name="odf-operator.openshift-storage")
```

Comment:
```suggestion
    odf_csv = OCP(kind="operator", resource_name="odf-operator", namespace=namespace
```

The namespace definition is on the line 221 so you will need to move it above this.

Reply:
This is not namespace but part of the operator name. The operator can be get from any namespace. It doesn't have in metadata the namespace mentioned. The operator is creating the namespace when installing it and therefore it is not in the openshift .

Reply:
metadata:
  creationTimestamp: "2022-06-21T19:06:33Z"
  generation: 1
  name: odf-operator.openshift-storage